source/docs/software/actuators/overview.rst

```diff
@@ -0,0 +1,68 @@
+Actuator Overview
+==================
+
+This section discusses the control of motors and pneumatics
+through speed controllers, solenoids and pneumatics, and
+their interface with C++ and Java WPILib.
+
+
+
+
+
+Speed controllers
+------------------------
+
+A speed controller is responsible on your robot for making motors move.
+For brushed DC motors such as CIMs or 775s, the speed controller regulates
+the voltage that the motor recieves, much like a light bulb. For
+brushless speed controllers such as the Spark MAX, the controller regulates
+the power delivered to each "phase" of the motor.
+
+.. hint:: 
```

Comment:
This may not be the place to suggest doing this to teams, at least not without supporting documentation.